include/c_types/path_dd_rt.h

```diff
@@ -1,8 +1,9 @@
 /*PGR-GNU*****************************************************************
-File: path_rtdd.h
+File: path_dd_rt.h
 
-Copyright (c) 2015 Celia Virginia Vergara Castillo
-Mail: vicky AT erosion.dev
+Generated with Template by:                                                                                             
+Copyright (c) 20xx pgRouting developers                                                                                 
+Mail: project@pgrouting.org   
```

Comment:
Please use `at` notation instead of @
Also, change the year to 2023
(same for other files generated using template)

Reply:
Okay